src/utils/util.js

```diff
@@ -204,7 +204,8 @@ const dispatchNotification = async (title, body, silent, handleClick) => {
     tag: body,
     icon: appIconURL,
     requireInteraction: false,
-    silent
+    silent,
+    data: {soundName}
```

Comment:
why not send data as the original argument in the message passing from the webapp? that way, if we change what's inside data because we want to add some more metadata/options we won't have to change all the signatures as you had to do.

Reply:
Ok I got it :)

Reply:
Ok I got it